src/app/components/header/header.component.html

```diff
@@ -10,29 +10,42 @@
     <nav class="nav-wrapper">
       <span *ngIf="!isBookingUrl" class="nav-text">Book flights</span>
 
-      <mat-form-field class="date-picker" appearance="outline">
-        <mat-label class="label">MM/DD/YYYY</mat-label>
-        <input matInput [matDatepicker]="picker" class="input" />
-        <mat-datepicker-toggle matIconSuffix [for]="picker">
-          <mat-icon matDatepickerToggleIcon class="icon">keyboard_arrow_down</mat-icon>
-        </mat-datepicker-toggle>
-        <mat-datepicker #picker></mat-datepicker>
+      <mat-form-field class="date-input" appearance="outline">
+        <mat-select
+          class="select"
+          value="MM/DD/YYYY"
+          (selectionChange)="setUserSettings($event.value, 'date')"
+        >
+          <mat-option class="option" value="MM/DD/YYYY">MM/DD/YYYY</mat-option>
+          <mat-option class="option" value="DD/MM/YYYY">DD/MM/YYYY</mat-option>
+          <mat-option class="option" value="YYYY/DD/MM">YYYY/DD/MM</mat-option>
+          <mat-option class="option" value="YYYY/MM/DD">YYYY/MM/DD</mat-option>
+        </mat-select>
       </mat-form-field>
 
       <mat-form-field class="currency-input" appearance="outline">
-        <mat-select class="select" value="EUR">
+        <mat-select
+          class="select"
+          value="USD"
+          (selectionChange)="setUserSettings($event.value, 'currency')"
+        >
           <mat-option class="option" value="EUR">EUR</mat-option>
           <mat-option class="option" value="USD">USD</mat-option>
           <mat-option class="option" value="RUB">RUB</mat-option>
```

Comment:
Please change values to lowercase (for convenience to use it in object)

Reply:
![image](https://github.com/Zmicer-Yafremau/airways/assets/94961194/e549bf9d-894f-4b18-a4b2-467e954828e3)
And we have 4 currencies. Can you make a special type for that?

Reply:
done